Pyramid.java

```diff
@@ -1,32 +1,55 @@
-/*
- * File: Pyramid.java
- * Name: 
- * Section Leader: 
- * ------------------
- * This file is the starter file for the Pyramid problem.
- * It includes definitions of the constants that match the
- * sample run in the assignment, but you should make sure
- * that changing these values causes the generated display
- * to change accordingly.
- */
-
-import acm.graphics.*;
-import acm.program.*;
-import java.awt.*;
-
-public class Pyramid extends GraphicsProgram {
-
-/** Width of each brick in pixels */
-	private static final int BRICK_WIDTH = 30;
-
-/** Width of each brick in pixels */
-	private static final int BRICK_HEIGHT = 12;
-
-/** Number of bricks in the base of the pyramid */
-	private static final int BRICKS_IN_BASE = 14;
-	
-	public void run() {
-		/* You fill this in. */
-	}
-}
-
+/*
+ * File: Pyramid.java

```

Comment:
Great solution. Now this is a scalable solution because now when I change BRICK_WIDTH, BRICK_HEIGHT and BRICK_IN_BASE your program still draws a pyramid at the center of the screen.